presto-tests/src/test/java/com/facebook/presto/tests/TestQuerySpillLimits.java

```diff
@@ -64,13 +63,6 @@ public void tearDown()
     public void testMaxSpillPerNodeLimit()
             throws Exception
     {
-        Map<String, String> properties = ImmutableMap.<String, String>builder()
```

Comment:
remove `//spill constantly` comment above.
Or replace with more appropriate one.

Edit:

Drop spill related configuration from `SESSION` and use approppriate setters on `FeaturesConfig` below as we already do for some parameters.


Edit2:

Do not set `AGGREGATION_OPERATOR_UNSPILL_MEMORY_LIMIT` it is unrelated to this test.